flink-runtime/src/main/java/org/apache/flink/runtime/operators/RegularPactTask.java

```diff
@@ -525,7 +525,9 @@ protected void run() throws Exception {
 				try {
 					FunctionUtils.closeFunction(this.stub);
 				}
-				catch (Throwable t) {}
+				catch (Throwable t) {
+					// do nothing
+				}
 			}
```

Comment:
The javadocs comment describe as to why we do a catch all and suppress all exceptions/errors. :)